_posts/2016/2016-12-20-cqrs-and-elasticsearch.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "CQRS and Elasticsearch"
```

Comment:
Is this headline set in stone? 
Maybe we can find a header that is even more SEO-friendly :)
check out: http://coschedule.com/headline-analyzer